lib/config.py

```diff
@@ -10,7 +10,11 @@ class Config(object):
 
     @classmethod
     def load(cls, filename):
-        cls.data = json.load(open(filename))
+        return cls.loads(open(filename).read())
+
+    @classmethod
+    def loads(cls, text_data):
+        cls.data = json.loads(text_data)
```

Comment:
I find the use of classmethod in this context surprising/confusing. Are you trying to make a singleton? If so, why? If not, why don't we use regular method here, maybe called from `__init__`? But I don't even understand why aren't we using `json.load` directly so probably never mind.